x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/styles.tsx

```diff
@@ -128,12 +141,24 @@ export const NodeShapeContainer = styled.div`
   height: ${NODE_HEIGHT}px;
 `;
 
-export const NodeShapeSvg = styled.svg`
+const nodeShadowStyle = css`
```

Comment:
Last status of active status was out of scope, so incase it was discussed otherwise and it will be removed, ignore my comments.

- Shadow is not similar to designs, in the design I notice the shadow is not visible at the top of the node. While in your implementation it is also appears at the top of the ellement.

- EUI provides `useEuiShadow('m')` which aligns to figma design, have you tried it?
- Node shadow is not enabled when item is active (using tabs)

<img width="228" height="224" alt="Screenshot 2025-07-27 at 14 26 58" src="https://github.com/user-attachments/assets/78342b2b-78c2-4b73-b437-68adcb1ff282" />